lib/pg_ha_migrations/safe_statements.rb

```diff
@@ -622,12 +594,6 @@ def safely_acquire_lock_for_table(*tables, mode: :access_exclusive, &block)
     end
   end
 
-  # since rails versions < 7.1 has bug which does not handle symbol for
-  # constraint name, we are converting name to string explicitly to solve that.
-  def raw_remove_check_constraint(table, name:, **options)
-    super(table, name: name.to_s, **options)
-  end
```

Comment:
As I was looking to organize methods and their tests, I discovered this unicorn. Rails 7.0 is EOL, so my solution was to just remove this method and Rails 7.0 from our gemspec / build matrix